pages/inspector/_meta.json

```diff
@@ -1,22 +1,27 @@
 {
   "start-using-inspector": "Quickstart: Get started with Inspector",
+  "inspector-installation-overview": "Inspector Installation Overview",
+  "configuring-inspector-sources": "Configuring Sources",
   "-": {
     "type": "separator"
   },
-  "inspector-overview": "Inspector Overview",
```

Comment:
We need to remember to re-direct this url to somewhere. It's in `/next.config.js` 

Reply:
I was going to ask you about that! I removed two pages. I will add them to `/next.config.js`

Reply:
Done!